### PR TITLE
chore(version): v1.0.0-next.1

### PR DESCRIPTION
**Pull request by dnys1**

### CODE REVIEW

**Review by Jordan-Nelson — APPROVED** [review]

File: packages/amplify_authenticator/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.0.0-next.1
+
+- Minor bug fixes and improvements
```
Comment (Jordan-Nelson) [inline packages/amplify_authenticator/CHANGELOG.md:3]: The authenticator does have one fix that I don't think made it into the previous release.

```suggestion
### Fixes
- fix(authenticator): remove navigator from tree for AuthenticatedState ([#2354](https://github.com/aws-amplify/amplify-flutter/pull/2354))
```

Comment (dnys1) [reply packages/amplify_authenticator/CHANGELOG.md:3]: Ah good catch 👍

File: packages/amplify/amplify_flutter/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+### Breaking Changes
+
+Three new categories have added support for Web + Desktop: API, Storage, and Analytics!
+
+See our [docs](https://docs.amplify.aws/lib/q/platform/flutter/) for guides on how to migrate to these new versions.
+
+### Features
+- feat(api): GraphQL Custom Request Headers ([#1938](https://github.com/aws-amplify/amplify-flutter/pull/1938))
+- feat(api): Subscription Reconnection ([#2074](https://github.com/aws-amplify/amplify-flutter/pull/2074))
+- feat(api): authorizationMode property for GraphQLRequest ([#2143](https://github.com/aws-amplify/amplify-flutter/pull/2143))
```
Comment (ragingsquirrel3) [inline packages/amplify/amplify_flutter/CHANGELOG.md:12]: nit/suggestion: Would it make sense to add a note about REST API interface changes? For appropriate PR link, I guess you would have to use https://github.com/aws-amplify/amplify-flutter/pull/2148.

File: packages/api/amplify_api/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+
+Initial preview release of the Dart rewrite!
+
+See our [docs](https://docs.amplify.aws/lib/q/platform/flutter/) for guides on how to migrate to this new version.
+
+### Fixes
+- fix(api): correct subscription error handling ([#2179](https://github.com/aws-amplify/amplify-flutter/pull/2179))
+- fix(api): fix model helper util on web ([#2270](https://github.com/aws-amplify/amplify-flutter/pull/2270))
+
+### Features
```
Comment (ragingsquirrel3) [inline packages/api/amplify_api/CHANGELOG.md:13]: same comment regarding REST API changes

**Review by ragingsquirrel3 — APPROVED** [review]
approved w 1 small non-blocking nit/question

Woooooh!

File: packages/amplify/amplify_flutter/CHANGELOG.md (hunk 2 above)
Comment (dnys1) [reply packages/amplify/amplify_flutter/CHANGELOG.md:15]: Done 👍

**Review by HuiSF — APPROVED** [review]
🚀✨

**Pull request merged**